src/Features/Core/Portable/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -57,19 +57,11 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
             var installerService = GetPackageInstallerService(document);
             var packageSources = searchNuGetPackages && symbolSearchService != null && installerService?.IsEnabled(document.Project.Id) == true
-                ? await installerService.TryGetPackageSourcesAsync(allowSwitchToMainThread: false, context.CancellationToken).ConfigureAwait(false)
```

Comment:
note: `allowSwitchToMainThread:` was always passed false.  so i was able to remove a bunch of code related to that that is just not needed.